acceptance/tests/facts/networking_facts.rb

```diff
@@ -0,0 +1,80 @@
+test_name 'FACT-1361 - C59029 networking facts should be fully populated'
+
+#
+# This test is intended to ensure that networking facts resolves
+# as expected in AIO across supported platforms.
+#
+
+skip_test "Networking fact test is confined to AIO" if @options[:type] != 'aio'
```

Comment:
this is targeted at master, so i don't think this is required.  even if merged down to stable, it'll still always be aio.  unless i'm missing something about network devices, where they aren't run with aio type?